src/java/org/apache/cassandra/utils/JMXServerUtils.java

```diff
@@ -227,18 +238,17 @@ private static MBeanServerForwarder configureJmxAuthorization(Map<String, Object
      * for configuring this.
      *
      * @param serverAddress the JMX server is bound to
-     * @param localOnly {@code true} if the JMX server only allows local connections; {@code false} if the JMX server
-     *                              allows the remote connections.
+     * @param options       {@code true} if the JMX server only allows local connections; {@code false} if the JMX server
```

Comment:
@smiklosovic The description of this variable needs to be updated for the options variable vs boolean before.